cli/main.c

```diff
@@ -1097,6 +1097,10 @@ static int log_parse(int argc, char **argv)
 		fprintf(stderr, "\nParsed log saved to %s.\n",
 			cfg.parsed_log_filename);
 
+	if (ret == ENOEXEC)
+		fprintf(stderr, "\nWARNING: The two input files have different version numbers.\n"
+				"The log file is parsed but the output file might contain errors.\n");
+
```

Comment:
It can be helpful to print the versions that mismatch.

Reply:
At this level there is no function to get the version numbers of log file and log-def file. 

Do you want me to add 2 functions like:
```
int switchtec_log_get_header(struct log_header *header)    //returns bin header info
int switchtec_log_def_get_header(struct log_def_header *header) //returns log-def header info
```
so we can print the versions from `header`?

I don't see many use cases for these 2 functions, but it doesn't hurt to add these.

Reply:
How about add an out structure parameter to switchtec_parse_log() to pass out the versions of log and def files?

Reply:
Good idea. Code is updated. Output looks like this:
```./switchtec log-parse switchtec-ram.log oslog-def-445.dat

Parsed log saved to log.txt.

WARNING: The two input files have different version numbers.
                FW Version      SDK Version
Log file:       0x06010d0e      0x00000000
Log def file:   0x06010d0e      0x06010d0e

The log file is parsed but the output file might contain errors.```